EIPS/eip-8192.md

```diff
@@ -0,0 +1,213 @@
+---
+eip: 8192
```

Comment:
```suggestion
eip: 7110
```
Assigning EIP number as first closed PR #7110.  Please also rename the filename.